src/intfuncs.c

```diff
@@ -620,6 +620,293 @@ Obj FuncINT_STRING ( Obj self, Obj string )
 
 /****************************************************************************
 **
+*F SmallInt Bitmask operations
+*
+*  Since all our functions have special handlers and no BODY, NLOC is unused,
+*  so we repurpose it.
+*
+*/
+
+#ifdef SYS_IS_64_BIT
+#define LENSTART_WIDTH 8
+#define WIDTH_WIDTH 6
+#define MAX_FIELDS 10
+#else
+#define LENSTART_WIDTH 8
+#define WIDTH_WIDTH 5
+#define MAX_FIELDS 6
+#endif
+
+static inline UInt get_start(UInt mask) {
+  return mask & ((1L << LENSTART_WIDTH) -1);
+}
+
+static inline UInt get_len(UInt mask) {
+  return mask >> LENSTART_WIDTH;
+}
+
+static inline UInt make_mask(UInt start, UInt len) {
+  return len << LENSTART_WIDTH | start;
+}
+
+static inline UInt get_width(UInt mask, UInt i) {
+  return (mask >> (WIDTH_WIDTH*(i-1)) & ((1 << WIDTH_WIDTH) -1));
+}
+
+static inline UInt set_width(UInt mask, UInt i, UInt width) {
+  return (((mask >> (WIDTH_WIDTH*i) << WIDTH_WIDTH) | width) << (WIDTH_WIDTH*(i-1))) | (mask & ((1L << (WIDTH_WIDTH*(i-1))) -1L));
+}
+
+static Obj doMaskGetter(Obj self, Obj data) {
+  UInt mask = NLOC_FUNC(self);
+   UInt start = get_start(mask);
+       UInt len = get_len(mask); 
```

Comment:
Funky formatting here.

Perhaps you might be willing to consider using `clang-format`?